src/kirby/components/floating-action-button/floating-action-button.component.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../scss/native-imports';
+
+// Ionic fab can be customized by using the css properties listed in the CSS Custom Properties section of the below docs page:
+// https://ionicframework.com/docs/api/fab-button
+
+.fab {
+    // using explicit evaluation #{} for the get-color function according to https://github.com/sass/sassc/issues/248
+    --background: #{get-color('kirby-primary')};
+    --box-shadow: 0 10px 20px -10px #{get-color('kirby-primary-shade')};
+    width: 64px;
```

Comment:
Question.... shouldn't the size be externally defined, in a variable?

Reply:
I agree with you, they could easily be defined externally. However, I did this in correlation to the style guide in Zeplin and no variables were used there. Furthermore, we need to define them in the code and we don't have that yet.

Reply:
think this is a great candidate for the guild meeting this morning